src/overlays/actors/ovl_En_Cne_01/z_en_cne_01.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_cne_01.h"
+#include "objects/object_cne/object_cne.h"
 
 #define FLAGS 0x00000019
```

Comment:
```suggestion
#define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_8 | ACTOR_FLAG_1)
```
I think we are starting to use this way